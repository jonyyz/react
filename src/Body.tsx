import * as React from "react";
import {
  createStyles,
  withStyles
} from "@material-ui/core/styles";

import Routes from "./Routes";

const styles = () => createStyles({
  container: {
    height: "100%",
    overflow: "hidden",
    display: "flex",
    flexDirection: "column",
    alignContent: "flex-start",
  }
});

interface Props {
  classes: any;
}

class Body extends React.Component<Props> {
  render() {
    const { classes } = this.props;

    return (
      <div className={classes.container}>
        <Routes />
      </div>
    )
  }
}

export default withStyles(styles)(Body);
