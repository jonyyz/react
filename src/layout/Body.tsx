import * as React from "react";
import {
  makeStyles,
  createStyles
} from "@material-ui/styles";

import Routes from "./Routes";

const useStyles = makeStyles(createStyles({
  container: {
    height: "100%",
    overflow: "hidden",
    display: "flex",
    flexDirection: "column",
    alignContent: "flex-start",
  }
}));

const Body: React.FunctionComponent = () => {
  const classes = useStyles({});

  return (
    <div className={classes.container}>
      <Routes />
    </div>
  );
}

export default Body;
