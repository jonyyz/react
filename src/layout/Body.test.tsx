import * as React from "react";
import { shallow } from "enzyme";

import Body from "./Body";

describe("Body", () => {
  test("should render", () => {
    const wrapper = shallow(<Body />);
    expect(wrapper).toMatchSnapshot();
  });
});
