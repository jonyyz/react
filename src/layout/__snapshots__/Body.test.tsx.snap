// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Body should render 1`] = `
<div
  className="makeStyles-container-1"
>
  <Routes />
</div>
`;
